Source/JavaScriptCore/API/JSContext.mm

```diff
@@ -259,11 +259,11 @@ - (JSVirtualMachine *)virtualMachine
 
 - (NSString *)name
 {
-    auto name = adopt(JSGlobalContextCopyName(m_context));
+    SUPPRESS_UNCOUNTED_ARG auto name = adopt(JSGlobalContextCopyName(m_context));
     if (!name)
         return nil;
 
-    return adoptCF(JSStringCopyCFString(kCFAllocatorDefault, name.get())).bridgingAutorelease();
+    SUPPRESS_UNCOUNTED_ARG return adoptCF(JSStringCopyCFString(kCFAllocatorDefault, name.get())).bridgingAutorelease();
```

Comment:
I also don't fully understand this. Is this working around a deficiency in our checker?

Reply:
cc @rniwa. This is a false positive. The static analysis is complaining about `name` being uncounted and unsafe.

Reply:
I filed rdar://145661220 and will add a FIXME comment.